pages/faqs.md

```diff
@@ -170,6 +170,10 @@ The Semantic Web and Linked Data technologies are centered on the question of in
 
 <a id="fewer_passwords"/>
 
+## Are all Solid compatible Pod providers and apps privacy preserving? 
+
+When software services are Solid compatible it means that you as a user are free to switch to a similar alternative and bring your data along. For example, if you wanted to switch messaging app you could bring your contacts and chat history with you to the new messaging app. 
```

Comment:
That's a non-answer though; it answers a different question than the one asked.

That said, "privacy-preserving" is a notion that requires an immense burden of proof, what we cannot guarantee in general (just like not every website on the Web if safe).